src/router/index.js

```diff
@@ -1,25 +1,31 @@
-import Vue from 'vue'
-import VueRouter from 'vue-router'
-import TheHome from '../views/TheHome.vue'
-import ResultsContainer from '../views/ResultsContainer.vue'
+import Vue from 'vue';
+import VueRouter from 'vue-router';
+import TheHome from '../views/TheHome.vue';
+import ResultsContainer from '../views/ResultsContainer.vue';
+import FindPark from '../views/FindPark.vue';
 
-Vue.use(VueRouter)
+Vue.use(VueRouter);
 
 const routes = [
   {
     path: '/',
     name: 'Home',
-    component: TheHome
+    component: TheHome,
   },
   {
     path: '/results',
     name: 'Results',
-    component: ResultsContainer
-  }
-]
+    component: ResultsContainer,
+  },
+  {
+    path: '/find-park',
```

Comment:
Question: Are we still using this path for `FindPark` if it is just being rendered as a child in `TheHome`?